client/verta/tests/unit_tests/test_deployed_model_class.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+""" Unit tests for the verta.deployment.DeployedModel class. """
+
+import os
+import pytest
+import responses
+
+from responses import matchers
+from requests import Session
+from requests.exceptions import RetryError, HTTPError
+from typing import Any, Dict, List
+from unittest import TestCase
+from unittest.mock import Mock, patch
+from urllib3 import Retry
+
+from verta.credentials import EmailCredentials
+from verta.deployment import DeployedModel
+from verta._internal_utils import http_session_utils
```

Comment:
```suggestion
import os
from typing import Any, Dict, List

import pytest
from requests import Session
from requests.exceptions import RetryError, HTTPError
import responses
from responses import matchers
from unittest import TestCase
from unittest.mock import Mock, patch
from urllib3 import Retry

from verta.credentials import EmailCredentials
from verta.deployment import DeployedModel
from verta._internal_utils import http_session_utils
```

[PEP 8](https://peps.python.org/pep-0008/#imports):

>  Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.